sky/task.py

```diff
@@ -471,8 +470,11 @@ def set_storage_mounts(
                     self.workdir is not None):
```

Comment:
Is `target == constants.SKY_REMOTE_WORKDIR` sufficient?  The RHS is a relative path starting with `~/`. Should we catch the case where `target` equals `/home/ubuntu/sky_workdir` assuming `ubuntu` is the username?

Reply:
Good catch! It is a bit tricky to catch the case. I added a TODO comment and leave it to the future. : )